bundles/org.openhab.binding.nobohub/src/main/resources/OH-INF/i18n/nobohub_no.properties

```diff
@@ -0,0 +1,71 @@
+# binding
+
+binding.nobohub.name = Glen Dimplex Nob� Hub Binding
+binding.nobohub.description = This is the binding for Glen Dimplex Nob� Hub.
+
+# thing types
+
+thing-type.nobohub.component.label = Komponent
+thing-type.nobohub.component.description = En komponent kan v�re en panelovn, bryter eller gulv termostat
+thing-type.nobohub.nobohub.label = Nob� Hub
+thing-type.nobohub.nobohub.description = Nob� Hub Bru Brinding
+thing-type.nobohub.zone.label = Sone
+thing-type.nobohub.zone.description = En sone kan inneholde flere flere Nob� enheter
+
+
+thing-type.nobohub.nobohub.channel.activeOverrideName.label = Global Aktiv Overstyring
+thing-type.nobohub.nobohub.channel.weekProfiles.label = Ukesprofiler
+thing-type.nobohub.component.channel.currentTemperature.label = N�v�rende Temperatur
+thing-type.nobohub.zone.channel.activeWeekProfileName.label = Aktiv Ukeprofil Navn
+thing-type.nobohub.zone.channel.activeWeekProfile.label = Aktiv Ukeprofil
+thing-type.nobohub.zone.channel.comfortTemperature.label = Komfort Temperatur
+thing-type.nobohub.zone.channel.ecoTemperature.label = Eco Temperatur
+thing-type.nobohub.zone.channel.currentTemperature.label = N�v�rende Temperatur
+thing-type.nobohub.zone.channel.calculatedWeekProfileStatus.label = Beregnet Ukeprofil Status
+
+# thing types config
+
+thing-type.config.nobohub.nobohub.serialNumber.label = Serialnummer
+thing-type.config.nobohub.nobohub.serialNumber.description = Nob� Hub serialnummer (12 tall)
+thing-type.config.nobohub.nobohub.hostName.label = Tjeneradresse
+thing-type.config.nobohub.nobohub.hostName.description = Tjener eller IP addresse til Nob� Hub
+thing-type.config.nobohub.nobohub.keepaliveInterval.label = Tidsintervall
+thing-type.config.nobohub.nobohub.keepaliveInterval.description = Tidsintervall (sekunder). Standardinstilling: 14
+thing-type.config.nobohub.zone.id.label = Id
+thing-type.config.nobohub.zone.id.description = Id for sone
+thing-type.config.nobohub.component.serialNumber.label = Serialnummer
+thing-type.config.nobohub.component.serialNumber.description = Serialnummer for komponent (12 tall)
+
+
+# channel types
+
+channel-type.nobohub.activeOverrideName-channel-type.label = Aktiv Overstyring
+channel-type.nobohub.activeOverrideName-channel-type.description = Modus p� aktiv overstyring
+channel-type.nobohub.activeOverrideName-channel-type.state.option.NORMAL = Normal
+channel-type.nobohub.activeOverrideName-channel-type.state.option.COMFORT = Komfort
+channel-type.nobohub.activeOverrideName-channel-type.state.option.ECO = Eco
+channel-type.nobohub.activeOverrideName-channel-type.state.option.Away = Borte
+channel-type.nobohub.eco-temperature-channel-type.label = Eco Temperatur
```

Comment:
There are some inconsistencies in this translation file. Please make sure to have same order, strings etc. as in original language. Ideally the files should have the same number of lines.